src/main/java/fi/hsl/common/metro/MetroStops.java

```diff
@@ -0,0 +1,71 @@
+package fi.hsl.common.metro;
+
+import com.typesafe.config.Config;
+import fi.hsl.common.config.ConfigParser;
+import fi.hsl.common.transitdata.PubtransFactory;
+import org.apache.pulsar.shade.com.google.common.collect.ArrayListMultimap;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+
+public class MetroStops {
```

Comment:
Is this class needed anywhere else than in `transitdata-metro-ats-parser`? If not, then this can be moved there.

Reply:
Good point. Moving that there.